specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -704,7 +704,7 @@
       ]
     },
     "Port": {
-      "description": "The port.",
+      "description": "The port exposed on the container group.",
```

Comment:
Is this accurate?

Reply:
@yolo3301 from what I am reading in the swagger, "ports" are the ports utilized in the container group while "containerPorts" refer to individual instances. Could you verify my references are correct?

Reply:
You're right. Ports under container group are the ports exposed with the public IP. Instance ports can be exposed for container-wise communication.